BibLaTeX.js

```diff
@@ -471,8 +471,9 @@ function encodeFilePathComponent(value) {
 			if(item.itemType == "book" && !creatorCheck(item,"author") &&
 			   creatorCheck(item, "editor")) type = "collection";
 
-			//biblatex recommends us to use mvbook for multi-volume books
-			if (type == "book" && item.volume) type = "mvbook"
+			//biblatex recommends us to use mvbook for multi-volume book
+			//i.e. a book with "# of vols" filled
+			if (type == "book" && item.numberOfVolumes) type = "mvbook"
```

Comment:
Logically, we should check that `item.numberOfVolumes > 1`, since we do sometimes import 1 into that field. However, that would break if someone entered something like "2 vols". So this is ok.